src/lib/section-categories.ts

```diff
@@ -61,3 +62,19 @@ export const isUniquePresent = (
 ) => {
   return isUnique(cat._id) && Object.keys(existingCats).includes(cat._id)
 }
+
+export const isBackMatterSection = (groupId: string): boolean => {
+  return groupId === 'MPSectionCategory:backmatter'
+}
+
+export const isBackMatterSectionCategoryExist = (parent: ManuscriptNode, category: string): boolean => {
+  let found = false;
+  parent.descendants((node: ManuscriptNode) => {
+    if (node.attrs.category === category) {
+      found = true;
+      return false;
```

Comment:
you could also return false if found == true to practically shot down the walk-through